source/refunding_payments/index.html.md.erb

```diff
@@ -182,70 +180,71 @@ For example, the response body for a refund would include:
 | success                  | The refund has been successfully processed.                                                                        |
-| error                    | It was not possible for the payment processor to make the refund.                                                  |
+| error                    | It was not possible for the payment service provider to make the refund.                                                  |
 
 <div style="height:1px;font-size:1px;">&nbsp;</div>
 
-To manage this, you must use <a
+To manage refunds in live environments, you must use <a
 href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/payment-id/find-payment-refund-by-id"
 target="blank">Find payment refund by ID</a> (link opens in new window) to
-check the processing status of the refund until it changes to either
-``success`` or ``error``.
+check the processing status of the refund, until it changes to either
+`success` or `error`.
 
 It will typically take 30 minutes for the status to change. You should not
 check the status more than once every 5 minutes.
 
 In the event of an error, GOV.UK Pay will not currently provide any more
 information. Please [contact
-us](/support_contact_and_more_information/#contact-us) if more information is
-required about why a refund failed.
+us](/support_contact_and_more_information/#contact-us) if you need more
+information required about why a refund failed.
 
-## Refunding from the GOV.UK Pay admin site
+## Refunding from the GOV.UK Pay admin tool
 
 You can use the [GOV.UK Pay admin
 site](https://selfservice.payments.service.gov.uk) to view transactions and
 issue refunds.
 
-Go to the __Transactions__ section of your accoun page to see a list of
-transactions. For example:
+Go to the __Transactions__ section to see a list of transactions. For example,
+in the following image::
 
 ![](images/transaction+list+image+4.png)
 
 In this list, you can select an individual payment to see details of that
 payment, including any previous refunds.
 
 For a successful payment with no previous refunds, you can use the red
-**Refund payment** button to carry out a full or partial payment.
+__Refund payment__ button to make a full or partial payment.
 
 ![](images/refund-payment-button.png)
 
-## Refund notifications
+## Refund email notifications
+
+You can use the GOV.UK Pay admin tool to [send email notifications to
+users](https://selfservice.payments.service.gov.uk/email-notifications).
 
-End users are automatically notified by email about successful payments.  This
-depends on settings you have entered in the self-service admin site. Users are
-not notified when a payment has either failed or is refunded - either manually
-or via the API. You should arrange to notify end users about payment failures
-and refunds as appropriate for your service.
+Users are not notified when a payment has either failed or is refunded. You
```

Comment:
we do send refund email notifications now. these are enabled by default on new accounts, and can be switched off in the same place as payment confirmation emails.